File: Remove Duplicates from Sorted Array/code.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int: #[1,1,1,1]
        j = 1
        for i in range(1, len(nums)):  #1,2
            if nums[i] != nums[i - 1]:
                nums[j] = nums[i]
                j += 1
                print(j)
        return j

File: Find Closest Number to Zero/code.cs
public class Solution {
    public int FindClosestNumber(int[] nums) {
            int temp;
            Array.Sort(nums);
            Array.Reverse(nums);
            int result = 0;
            for (int i = 0; i < nums.Length - 1; i++)
            {
                if (nums[i] >= 0 && nums[i + 1] <= 0)
                {
                    if (nums[i] <= nums[i + 1] * (-1)) {
                        result = nums[i];
                    }
                    else {
                        result = nums[i + 1];
                    }
                }
                else if (nums[0] < 0) {
                    result = nums[0];
                }
                else if (nums[nums.Length-1] > 0) {
                    result = nums[nums.Length-1];
                }
            }
            return result;
    }
}
